src/defaults/html.pipe.js

```diff
@@ -61,6 +61,10 @@ const htmlpipe = (cont, payload, action) => {
     .when(esi) // flag ESI when there is ESI in the response
     .error(selectStatus(production()));
 
+
+  // pipe.attach.before(foo, 'fetch');
```

Comment:
I suggest taking the function as a second argument…much more ergonomic…

Reply:
Oh and are these supposed to be comments?

Reply:
Removed them.